0000-template.md

```diff
@@ -1,97 +1,126 @@
 
-# Stakeholders
-[stakeholders]: #stakeholders
+If successful with CBRS, we see this expanding into WiFi (akin to a decentralized version of Google’s project Orion) as well as other, non-CBRS 5G bands for urban densification of cellular networks. 
 
-* Who is affected by this HIP?
+Aside from the required modifications to the miner software itself, enabling this experiment also requires a number of changes to the Helium blockchain. Specifically: 
 
-* How are we soliciting feedback on this HIP from these stakeholders? Note that
-  they may not be watching the HIPs repository or even aren't directly active in
-  the Helium Community Slack channels.
+- Introduce a way to redeem Data Credits towards passing data using a variety of RF protocols 
+- Introduce new Proof-of-Coverage model to suit the use case of LTE offload to CBRS, ideally in a fashion that would be portable to other wireless protocols that can be used for LTE offload (i.e. other non-CBRS 5G-friendly bands as well as WiFi)   
 
-# Detailed Explanation
-[detailed-explanation]: #detailed-explanation
+## Redeeming Data Credits for 5G, LTE and Wi-Fi
 
-- Introduce and explain new concepts.
+To make it possible to redeem Data Credits towards different types of wireless data, it is suggested that we introduce a chain variable that will function as a conversion ratio of DC to data packets as a function of: 
 
-- It should be reasonably clear how the proposal would be implemented.
+- Types of wireless protocol i.e. LTE, 5G, Wi-Fi etc. 
+- Specific location of the access point transmitting the packets, since data prices vary per geography
 
-- Provide representative examples that show how this proposal would be commonly
-  used.
+In the future, as we grow and expand the network, we may want to have different conversion ratios, but for the purposes of the initial contemplated pilot, we’d like to focus on introducing just a conversion ratio that would denominate economics of a CBRS network. To figure out the initial cost of CBRS Data Credit we looked at the following guiding factors: 
 
-- Corner cases should be dissected by example.
+- Cheap prepaid LTE data plans in US sell for $1.50/Gb (Mint Mobile 15Gb bundle) 
```

Comment:
Maybe add a reference to this price point? I am unable to find it. The 15GB/month plan costs $45 ($3/GB) when I check https://www.mintmobile.com/plans/